test/functional/test_framework/wallet_util.py

```diff
@@ -119,6 +134,44 @@ def test_address(node, address, **kwargs):
         elif addr_info[key] != value:
             raise AssertionError("key {} value {} did not match expected value {}".format(key, addr_info[key], value))
 
+
+class MiniWallet():
```

Comment:
do you think this should live in wallet_util (the docstring says "Useful util functions for testing the wallet") or should it be in its own module?

Reply:
I think a new `wallet.py` makes sense. Any objections?

Reply:
No objections!